src/molecules/formfields/shared/formfields.module.scss

```diff
@@ -179,18 +181,20 @@ $checkbox-border-color: color('neutral-5');
     display: flex;
   }
 
-  &-day,
-  &-mon {
-    border-top-right-radius: 0;
-    border-bottom-right-radius: 0;
+  &-input {
+    width: calc(100% / 3);
 
-    border-right: 1px solid color('neutral-4');
-  }
+    input {
+      width: 100%;
+    }
+
+    &:not( :last-child ) {
```

Comment:
can we avoid using `:not` selector here? maybe make all of these styles part of the `datefield-input` and then the `:last-child` has specific styles? it's more verbose, but makes our styles more flexibile